tests/end-to-end/director/application_api_test.go

```diff
@@ -984,6 +984,82 @@ func TestTenantSeparation(t *testing.T) {
 
 }
 
+func TestApplicationsForRuntime(t *testing.T) {
+	//Create first app in first tenant
+	//Given
+	ctx := context.Background()
+	inputApp := generateSampleApplicationInput("app1")
+	appInputGQL, err := tc.graphqlizer.ApplicationInputToGQL(inputApp)
+	require.NoError(t, err)
+	createReq := fixCreateApplicationRequest(appInputGQL)
+	firstApplication := graphql.Application{}
+
+	//WHEN
+	err = tc.RunQuery(ctx, createReq, &firstApplication)
+
+	//THEN
+	require.NoError(t, err)
+	assert.NotEmpty(t, firstApplication.ID)
+	defer deleteApplication(t, firstApplication.ID)
+
+	//Create second firstApplication
+	//GIVEN
+	inputApp.Name = "app2"
+	appInputGQL, err = tc.graphqlizer.ApplicationInputToGQL(inputApp)
+	require.NoError(t, err)
+	createReq = fixCreateApplicationRequest(appInputGQL)
+	secondApplication := graphql.Application{}
+
+	//WHEN
+	err = tc.RunQuery(ctx, createReq, &secondApplication)
+
+	//THEN
+	require.NoError(t, err)
+	assert.NotEmpty(t, secondApplication.ID)
+	defer deleteApplication(t, secondApplication.ID)
+
+	//create app in different tenant
+	//GIVEN
+	tenantName := "completely-another-tenant"
+	appInputGQL, err = tc.graphqlizer.ApplicationInputToGQL(inputApp)
+	require.NoError(t, err)
+	createReq = fixCreateApplicationRequest(appInputGQL)
+	createReq.Header["Tenant"] = []string{tenantName}
+	application := graphql.Application{}
+
+	//WHEN
+	err = tc.RunQuery(ctx, createReq, &application)
+
+	//THEN
+	require.NoError(t, err)
+	assert.NotEmpty(t, application.ID)
+	defer deleteApplicationInTenant(t, application.ID, tenantName)
+
+	// get all application withing tenant
+	//GIVEN
+	tenantApplications := []*graphql.Application{&firstApplication, &secondApplication}
+	request := gcli.NewRequest(
+		fmt.Sprintf(
+			`query {
+  			result: applicationsForRuntime(runtimeID: "%s", first:%d, after:"%s") { 
+					%s 
+				}
+			}`,
+			"rtm-id", 2, "next", tc.gqlFieldsProvider.ForApplicationPage(),
+		))
+	applicationPage := graphql.ApplicationPage{}
+
+	fmt.Println(request)
+
+	//WHEN
+	err = tc.RunQuery(ctx, request, &applicationPage)
+
```

Comment:
Add `saveQueryInExamples(t, request.Query(), "query applications for runtime")` to generate example for this request